docs/src/index.md

```diff
@@ -0,0 +1,31 @@
+# Light Graphs
+
+The goal of *LightGraphs.jl* is to mirror the functionality of robust network and graph analysis libraries such as [NetworkX](http://networkx.github.io) while being simpler to use and more efficient than existing Julian graph libraries such as [Graphs.jl](https://github.com/JuliaLang/Graphs.jl). It is an explicit design decision that any data not required for graph manipulation (attributes and other information, for example) is expected to be stored outside of the graph structure itself. Such data lends itself to storage in more traditional and better-optimized mechanisms.
```

Comment:
This is a foundational manifesto that we have shed with maturity. We are no longer defined by our relationship to NetworkX and Graphs.jl. We have our own fate in our own hands! Long Live LightGraphs!